src/m365/spfx/commands/spfx-doctor.ts

```diff
@@ -603,16 +605,27 @@ class SpfxDoctorCommand extends AnonymousCommand {
       });
   }
 
+  private checkGulpCli(prerequisites: SpfxVersionPrerequisites, fixes: string[], logger: Logger): Promise<void> {
+    return this
+      .getPackageVersion('gulp-cli', PackageSearchMode.GlobalOnly, HandlePromise.Continue, logger)
+      .then((gulpCliVersion: string): void => {
+        if (gulpCliVersion) {
+          this.checkStatus('gulp-cli', gulpCliVersion, prerequisites.gulpCli, OptionalOrRequired.Required, fixes, logger);
+        }
+        else {
+          logger.log(this.getStatus(CheckStatus.Failure, `gulp-cli not found`));
+          fixes.push(prerequisites.gulpCli.fix);
+        }
+      });
+  }
+
   private checkGulp(prerequisites: SpfxVersionPrerequisites, fixes: string[], logger: Logger): Promise<void> {
```

Comment:
Because we don't use prerequisites here, let's remove them from the method's signature to avoid unnecessary arguments